rest/replicator_test.go

```diff
@@ -1204,3 +1204,156 @@ func TestActiveReplicatorPushAndPullConflict(t *testing.T) {
 		})
 	}
 }
+
+// TestActiveReplicatorPushBasicWithInsecureSkipVerify:
+//   - Starts 2 RestTesters, one active, and one passive with TLS enabled and SgrTlsSkipVerify set to true.
+//   - Creates a document on rt1 which can be pushed by the replicator.
+//   - Publishes the REST API on a httptest server for the passive node (so the active can connect to it)
+//   - Uses an ActiveReplicator configured for push to start pushing changes to rt2.
+func TestActiveReplicatorPushBasicWithInsecureSkipVerifyEnabled(t *testing.T) {
+	if base.GTestBucketPool.NumUsableBuckets() < 2 {
+		t.Skipf("test requires at least 2 usable test buckets")
+	}
+	defer base.SetUpTestLogging(base.LevelDebug, base.KeyHTTP, base.KeySync, base.KeyChanges, base.KeyCRUD, base.KeyBucket)()
+
+	// Passive
+	tb2 := base.GetTestBucket(t)
+	defer tb2.Close()
+	rt2 := NewRestTester(t, &RestTesterConfig{
+		TestBucket: tb2,
+		DatabaseConfig: &DbConfig{
+			Users: map[string]*db.PrincipalConfig{
+				"alice": {
+					Password:         base.StringPtr("pass"),
+					ExplicitChannels: base.SetOf("alice"),
+				},
+			},
+			Unsupported: db.UnsupportedOptions{
+				SgrTlsSkipVerify: true,
+			},
```

Comment:
I don't think this needs to be set, `rt2` is the receiver of the request.

It's the connecting client's choice to skip verification (which is determined by the `InsecureSkipVerify: true` in the ActiveReplicator config down below)

Reply:
Makes sense, this is not required and has been removed.